config/config.exs

```diff
@@ -1,5 +1,7 @@
 use Mix.Config
 
+config :logger, backends: []
```

Comment:
This is to prevent us needing to capture all of the info logs in every test. Without this the test results contain all of the log messages.